CHANGES/1081.feature

```diff
@@ -0,0 +1,4 @@
+
+Implement ``__repr__`` for C Extension classes in C.
```

Comment:
Don't forget that the change log isn't supposed to look like a TODO-list or a Git log. It's supposed to reference things that have already changed compared to the previous release. Therefore, the use of imperative mood is incompatible with this document type.
The past tense or present simple works best, when explaining *to the end-users* how the changes affect *them*. This is your audience, not contributors.
```suggestion
Implemented :py:meth:`~object.__repr__` interface in the C Extension-based :py:class:`~multidict.Multidict` implementation.
```

Reply:
I've applied past tense.

Sorry, I disagree. with the other proposed changes.  A link to `__repr__` doesn't add too much for the reader. A link to `MultiDict` is not complete, the PR modifies reprs for all `multidict` classes except `istr`. Let's keep the text simple.

Reply:
Marking as resolved.